src/template-tags/tickets.php

```diff
@@ -332,17 +349,84 @@ function tribe_tickets_get_template_part( $slug, $name = null, array $data = nul
 	// loop through templates, return first one found.
 	foreach ( $templates as $template ) {
```

Comment:
There are going to be at least 2 elements in `$templates` here - `$slug.php` and `$slug-$name.php` and we look at both in the loop. Let's say the plugin ships with a `$slug.php` template and the theme has a `$slug-name.php` template: won't the `file_exists()` test path in both cases?

So, if `$echo` is true, doesn't that mean we're going to print both templates? If we don't echo it doesn't matter, because `$html` will be overwritten on each iteration - but maybe that suggests there is a structural issue and the decision to echo should be moved out of the loop and down to the same place as where we sometimes return the `$html`.